Hearthstone Deck Tracker/Hearthstone/Player.cs

```diff
@@ -32,6 +33,8 @@ public Player(IGame game, bool isLocalPlayer)
 		public bool IsLocalPlayer { get; }
 		public int SpellsPlayedCount { get; private set; }
 		public bool IsPlayingWhizbang { get; set; }
+		public int PogoHopperPlayedCount {get; private set;}
```

Comment:
This class is already scoped to either the local or the opposing player. You only need one counter.